specs/reporters/info.cpp

```diff
@@ -376,6 +376,43 @@ go_bandit([]() {
       });
     });
 
+    describe("a report with timing", [&]() {
+      before_each([&]() {
+        reporter.reset(new reporter::info(stm, formatter, true, colorizer));
+        reporter->test_run_starting();
+        reporter->test_run_starting();
+        reporter->context_starting("context");
+        reporter->it_starting("report timing");
+      });
+      it("shows a passing test as passed with timing", [&]() {
+        reporter->it_succeeded("passes");
+        reporter->context_ended("context");
+        reporter->test_run_complete();
+
+        AssertThat(output(), Contains("[ PASS ] it passes (0.0"));
+        AssertThat(output(), Contains(" seconds)"));
+      });
+
+      it("shows a failing test as failed with timing", [&]() {
+        bd::assertion_exception exception("assertion failed!", "some_file", 123);
+        reporter->it_failed("fails", exception);
+        reporter->context_ended("context");
+        reporter->test_run_complete();
+
+        AssertThat(output(), Contains("[ FAIL ] it fails (0.0"));
```

Comment:
Since I am only checking coarsely, I don't *know* why the CI fails here (and below), but it looks like a problem with uninitialized values.